src/style.css

```diff
@@ -0,0 +1,100 @@
+* {
+  box-sizing: border-box;
+}
+
+:root {
+  --linear-gradient: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  --br-solid: 1px solid #e1e7ed;
+}
+
+@font-face {
+  font-family: "Avenir-Book";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 700;
+}
+
+.search {
+  margin-top: 20px;
+  position: relative;
+  background: #fff;
+}
+
+.big_input {
+  font-family: "Avenir", sans-serif;
+  height: 70px;
+  width: 100%;
+  padding-left: 62px;
+  font-size: 16px;
+  line-height: 22px;
+  display: flex;
+  border: var(--br-solid);
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  outline: none;
+}
+
+.small_input {
+  font-family: "Avenir", sans-serif;
+  height: 42px;
+  width: 100%;
+  padding-left: 33px;
+  font-size: 14px;
+  line-height: 19px;
+  display: flex;
+  border: var(--br-solid);
+  border-radius: 4px;
+  outline: none;
+}
```

Comment:
Some styles are same for both - Do not repeat your self. Create separate class for this, use modifire for difference
![2023-01-25_16h09_38](https://user-images.githubusercontent.com/61904995/214585515-cbf041c8-c099-44ad-9f10-41ad96c1d447.png)